script/tool/lib/src/common/plugin_command.dart

```diff
@@ -174,60 +190,82 @@ abstract class PluginCommand extends Command<void> {
     _shardCount = shardCount;
   }
 
-  /// Returns the root Dart package folders of the plugins involved in this
-  /// command execution.
-  // TODO(stuartmorgan): Rename/restructure this, _getAllPlugins, and
-  // getPackages, as the current naming is very confusing.
-  Stream<Directory> getPlugins() async* {
+  /// Returns the set of plugins to exclude based on the `--exclude` argument.
+  Set<String> _getExcludedPackages() {
```

Comment:
`getExcludedPackagePaths`? `getExcludedPackageNames`?

Reply:
Yep, changed to getExcludedPackageNames. ("name" is actually itself a bit of a weird concept currently in the tool code currently, due to the retro-fitting of federated packages that happened at some point; I would like to someday go back and clean that up. But it's definitely better than just "package".)